Sources/SwiftMemcache/MemcachedResponseDecoder.swift

```diff
@@ -120,8 +119,24 @@ struct MemcachedResponseDecoder: NIOSingleStepByteToMessageDecoder {
 
         case .dataLengthOrFlag(let returnCode):
             if returnCode == .VA {
-                // TODO: Implement decoding of data length
-                fatalError("Decoding for VA return code is not yet implemented")
+                guard let dataLength = buffer.readInteger(as: UInt64.self) else {
```

Comment:
Is this correct? Is the data length encoded as a `UInt64` or rather written out as text?

Reply:
you are correct, it is written out as text, so I will make that change as well!